pkg/operator/resource/resourceapply/core.go

```diff
@@ -186,12 +186,12 @@ func SyncConfigMap(client coreclientv1.ConfigMapsGetter, recorder events.Recorde
 		source.Namespace = targetNamespace
 		source.Name = targetName
 		source.ResourceVersion = ""
-		source.OwnerReferences = []metav1.OwnerReference{}
+		source.OwnerReferences = ownerRefs
```

Comment:
can you do `source.SetOwnerReferences` ? (I believe that is part of `runtime.Object`)